molo/forms/tests/test_utils.py

```diff
@@ -237,6 +253,14 @@ def setUp(self):
             ),
             required=True
         )
+        self.hidden_field = MoloFormField.objects.create(
+            page=self.form,
+            sort_order=5,
+            default_value='cat',
+            label='Your least favourite animal',
+            field_type='hidden',
+            required=True
+        )
```

Comment:
Why is this needed? You don't use this variable in any of this classes tests

Reply:
@KaitCrawford Had to add the hidden form field in every test since I know the skip logic code is not the easiest to refactor, to be sure everything works as to be expected in every scenario
```
    self.assertEqual(
            self.paginator.page(1).object_list, [
                self.hidden_field,  # <- Hidden form field
                self.paginator.page(1).object_list[1],   # <- Dynamically added unsaved article form field instance
                self.first_field
            ],
    )
```

Reply:
ok. I think I understand now